enhancements/operator-dev-doc.md

```diff
@@ -0,0 +1,269 @@
+It also provides information about developing with OpenShift operators and the OpenShift release payload. 
+When updating READMEs in core OpenShift repositories, we realized there is overlap of content.
+We've created this document to serve as a common link for READMEs to answer `What is an Operator?` and 
+`How do I build/update/test/deploy this code?`.
+
+Many OpenShift Operators were built using [openshift/library-go's framework](https://github.com/openshift/library-go/tree/master/pkg/operator)
+and also utilize [openshift/library-go's alpha-build-machinery](https://github.com/openshift/library-go/tree/master/alpha-build-machinery), a 
+collection of building blocks, Makefile targets, helper scripts, and other build-related machinery.  As a result, there are common methods 
+for building, updating, and developing these operators.  When adding features or investigating bugs, you may need to swap out a 
+component layer of a release payload for a locally built one.  Again, the process for doing this is shared among OpenShift operators, as well
+as the process of updating CVO-managed deployments in a cluster.  
+
+### What is an OpenShift ClusterOperator?
+OpenShift deploys many operators on Kubernetes. Core OpenShift operators are ClusterOperators (CO). You can list them with this:
+```console
+$ oc get clusteroperators
+```
+To get a description of a CO run
+```console
+$ oc describe co/clusteroperatorname
+```
+Operators are controllers for a 
+[Custom Resource](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/).  Operators automate many tasks
+in application management, such as deployments, backups, upgrades, leader election, reconciling resources, etc.
+
+The [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) in OpenShift can be viewed with this:
+```console
+$ oc get crds
+
+For details on a particular CRD (ex: openshiftapiservers.operator.openshift.io):
+$ oc explain openshiftapiservers.operator.openshift.io
+```
+For more information, check out [the Operator pattern in Kubernetes](https://kubernetes.io/docs/concepts/extend-kubernetes/operator/).    
+
+While ClusterOperators manage individual components (Custom Resources), the 
+[Cluster Version Operator](https://github.com/openshift/cluster-version-operator) (CVO) is responsible for managing the ClusterOperators.
+
+### What is an Operand?
+The component that an operator manages is its operand.  For example, the cluster-kube-controller-manager-operator CO manages the 
+cluster-kube-controller-manager component running in OpenShift. An operator can be responsible for multiple components.  For example, 
+the CVO manages all of the COs (in this way ClusterOperators are also operands). 
+
+### What is an OpenShift release image?
+An OpenShift release image is comprised of layers.  Each layer digest is a single component image.  This includes operator
```

Comment:
Comprised of multiple images I thought.  just snip down to "to get a list of the components" for now and some else can add more detail.

Reply:
ok